static-assets/themes/cstudioTheme/css/forms-default.css

```diff
@@ -155,6 +155,15 @@ div.yui-calcontainer.cstudio-form-field-calendar {
     margin-bottom: 10px;
 }
 
+[role="dialog"] 
```

Comment:
Feels unnecessarily specific. Can you simplify?

Reply:
Done